diff --git a/lib/typescript.js b/lib/typescript.js
index fa7c62f9a01aa8d0be1aa34bac06b2ef0b2cebeb..bc04b4623fd4f5395b04fe565dae79d906847554 100644
--- a/lib/typescript.js
+++ b/lib/typescript.js
@@ -16116,7 +16116,7 @@ function isInternalDeclaration(node, sourceFile) {
 // src/compiler/utilities.ts
 var resolvingEmptyArray = [];
 var externalHelpersModuleNameText = "tslib";
-var defaultMaximumTruncationLength = 160;
+var defaultMaximumTruncationLength = 3000;
 var noTruncationMaximumTruncationLength = 1e6;
 function getDeclarationOfKind(symbol, kind) {
   const declarations = symbol.declarations;
